features/step_definitions/sample_file_steps.rb

```diff
@@ -0,0 +1,219 @@
+Given(/^a demo directory with a smelly file$/) do
+  contents = <<-EOS
+class Dirty
```

Comment:
I dont doubt that this works well, but i would rather use a library than monkeypatch string.;)